<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PMSExp</string>
  </property>
  <widget class="QWidget" name="m_central_widget">
   <layout class="QVBoxLayout" name="m_main_layout">
    <item>
     <widget class="QStackedWidget" name="m_stacked_widget">
      <widget class="QWidget" name="m_upload_page">
       <layout class="QVBoxLayout" name="m_upload_layout">
        <item>
         <widget class="QGroupBox" name="m_upload_box">
          <property name="title">
           <string>Chargement des données</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="gui::widgets::UploadWidget" name="m_ex_uploader" native="true">
             <property name="info" stdset="0">
              <string>Charger les données expérimentales</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="m_line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="gui::widgets::UploadWidget" name="m_th_uploader" native="true">
             <property name="info" stdset="0">
              <string>Charger les données théoriques</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="m_status_box">
          <property name="title">
           <string>Statut en cours</string>
          </property>
          <layout class="QVBoxLayout" name="m_item_layout">
           <item>
            <widget class="gui::widgets::CheckWidget" name="m_ex_item" native="true">
             <property name="info" stdset="0">
              <string>État de chargement des données expérimentales</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="gui::widgets::CheckWidget" name="m_th_item" native="true">
             <property name="info" stdset="0">
              <string>État de chargement des données théoriques</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_valid_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Valider</string>
          </property>
          <property name="icon">
           <iconset theme="system-run">
            <normaloff>.</normaloff>.</iconset>
          </property>
          <property name="shortcut">
           <string>Ctrl+R</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="m_statistic_page">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="gui::widgets::StatisticWidget" name="widget" native="true"/>
        </item>
        <item>
         <widget class="QPushButton" name="m_return_button">
          <property name="text">
           <string>Retour</string>
          </property>
          <property name="icon">
           <iconset theme="go-previous"/>
          </property>
          <property name="shortcut">
           <string>Ctrl+S</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="m_status_bar"/>
  <widget class="QToolBar" name="m_tool_bar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_action_quit"/>
   <addaction name="separator"/>
   <addaction name="m_action_reset"/>
  </widget>
  <action name="m_action_quit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="toolTip">
    <string>Quitte l'application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="m_action_reset">
   <property name="icon">
    <iconset theme="view-refresh"/>
   </property>
   <property name="text">
    <string>Réinitialiser</string>
   </property>
   <property name="toolTip">
    <string>Réinitialiser les fichiers chargés.</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>gui::widgets::CheckWidget</class>
   <extends>QWidget</extends>
   <header>gui/widgets/CheckWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>gui::widgets::UploadWidget</class>
   <extends>QWidget</extends>
   <header>gui/widgets/UploadWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>gui::widgets::StatisticWidget</class>
   <extends>QWidget</extends>
   <header>gui/widgets/StatisticWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
